misc/testerina/modules/testerina-core/src/main/ballerina/annotations.bal

```diff
@@ -22,13 +22,15 @@
 # + before - Function to be run before the test is run
 # + after - Function to be run after the test is run
 # + dependsOn - A list of functions the test function depends on and will be run before the test
+# + serialExecution - Flag to enable/disable parallization for a test function
```

Comment:
But isn't parallalization off by default?

Reply:
> But isn't parallalization off by default?
Only in the initial phase. Parallel execution will be enabled by default once we have passed the experimental phase. So we control that using the `--workers` flag which is set to 1 by default and later we would increase the number.

Reply:
AFAIR this variable name was discussed during design discussion and decided this way because serial execution is the default behavior. So, the flag should represent the default behavior.

@azinneera AFAIK during the discussions what we decided was that enabling parallel execution should be a conscious decision by user. Didn't we?

Reply:
> AFAIR this variable name was discussed during design discussion and decided this way because serial execution is the default behavior. So, the flag should represent the default behavior.
> 
> @azinneera AFAIK during the discussions what we decided was that enabling parallel execution should be a conscious decision by user. Didn't we?

Referred to the [meeting note](https://docs.google.com/document/d/1f9VpdyAXvPNL_jfmQdcAiG_y_POyGVsLmajmvWBX4Uk/edit#heading=h.e3f5luw6xwr1) and we have decided the other way around.

Reply:
AFAIR after the meeting this suggestion was given. That's why it is not recorded in the meeting notes. Earlier implementation had this `parallelizable` flag. Ref :- https://github.com/ballerina-platform/ballerina-lang/pull/40279/commits/10fbb84f68a5bdf5d72e5d25275024223a61acdf#diff-45e87dbd32b1175c366776df451bd7318454107dcc73505e54046a54f930affcR43 .